### ISSUE TITLE
Add Dagshub Logger & MLflow Tracking

### ISSUE THREAD

**Issue by nirbarazida** [jinensetpal/panoptic-deeplab#16]
Log the following metrics with MLflow Tracking (Train, validation, test sets):
1. Panoptic loss
2. Mean IoU
3. Average precision (AP) 
4. Panoptic quality (PQ)

Log the following metrics with DagsHub Logger (Test set):
1. Panoptic loss
2. Mean IoU
3. Average precision (AP) 
4. Panoptic quality (PQ)

Log all hyperparameters using both loggers.
